### PR TITLE
Add a disconnect button

### PR DESCRIPTION
**Pull request by g-ortuno**
So that we can simulate a Peripheral disconnecting. Also use getConnectedDevices to count the number of devices connected to the GATT Server.

### CODE REVIEW

**Comment by g-ortuno** [comment]
@jyasskin PTAL

**Comment by jyasskin** [comment]
LGTM. I assume this is to help investigate https://github.com/WebBluetoothCG/web-bluetooth/issues/192?

**Comment by g-ortuno** [comment]
device-2015-12-10-172834

**Comment by g-ortuno** [comment]
yup

File: app/src/main/java/io/github/webbluetoothcg/bletestperipheral/Peripheral.java [hunk 1]
```diff
@@ -292,7 +310,7 @@ private void resetStatusViews() {
 
   private void updateConnectedDevicesStatus() {
     final String message = getString(R.string.status_devicesConnected) + " "
-        + mBluetoothDevices.size();
+        + mBluetoothManager.getConnectedDevices(BluetoothGattServer.GATT).size();
```
Comment (beaufortfrancois) [inline app/src/main/java/io/github/webbluetoothcg/bletestperipheral/Peripheral.java]: Quick question as I have not been able to test this patch: Why don't we use `mBluetoothDevices` there?

Comment (g-ortuno) [inline app/src/main/java/io/github/webbluetoothcg/bletestperipheral/Peripheral.java]: The platform already offers a way to get the number of devices connected so it didn't make much sense to keep my own list.

**Pull request merged**